app/app.rb

```diff
@@ -12,19 +14,23 @@ class MakersBnb < Sinatra::Base
 
   set :session_secret, 'very SuPER DUPeR secret ha ha no hack please'
 
-
   get '/' do
     erb :index
   end
 
-
   get '/space' do
```

Comment:
We would suggest to change this in 'spaces/new' so to have a dedicated route to the 'new space' form.